docs/sops/serverless-controller-knative-up.md

```diff
@@ -0,0 +1,149 @@
+# OpenShift Serverless Knative Serving Control Plane Readiness
```

Comment:
Are you planning to add anything for Eventing too?

Reply:
Not right now. This SOP is needed for RHODS, but they don't use Eventing.